client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.tsx

```diff
@@ -166,25 +154,9 @@ const ProfessionalEmailCard = ( {
 
 	const onTitanFormReturnKeyPress = noop;
 
-	const priceWithInterval = (
-		<PriceWithInterval
-			intervalLength={ intervalLength }
-			cost={ titanMailProduct?.cost ?? 0 }
-			currencyCode={ currencyCode ?? '' }
-			hasDiscount={ isEligibleForFreeTrial }
-		/>
-	);
-
 	professionalEmail.onExpandedChange = onExpandedChange;
 	professionalEmail.priceBadge = (
-		<>
-			{ isDomainEligibleForTitanFreeTrial( domain ) && (
```

Comment:
I'm checking `isEligibleForFreeTrial` instead of `isDomainEligibleForTitanFreeTrial()` in the new component. I think it's more consistent since we already use `isEligibleForFreeTrial` for `PriceWithInterval`.